src/app/pages/record/add-event/add-event.component.html

```diff
@@ -1,30 +1,30 @@
 <p>Add event</p>
-<form>
+<form [formGroup]="form">
   <mat-form-field>
     <mat-label>Choose category</mat-label>
-    <mat-select>
+    <mat-select formControlName="category">
       <mat-option *ngFor="let category of data" [value]="category.id">{{category.name}}</mat-option>
     </mat-select>
-    <mat-error>Field is required</mat-error>
+    <mat-error *ngIf="form.get('category')?.errors?.['required']">Field is required</mat-error>
   </mat-form-field>
   <p>Choose type</p>
-  <mat-radio-group>
+  <mat-radio-group formControlName="type">
     <mat-radio-button value="income">Income</mat-radio-button>
     <mat-radio-button value="outcome">Outcome</mat-radio-button>
   </mat-radio-group>
   <mat-form-field>
     <mat-label>Enter sum</mat-label>
-    <input matInput>
-    <mat-error>Field is required</mat-error>
-    <mat-error>Field is invalid</mat-error>
+    <input matInput formControlName="amount">
+    <mat-error *ngIf="form.get('amount')?.errors?.['required']">Field is required</mat-error>
+    <mat-error *ngIf="form.get('amount')?.errors?.['pattern']">Field is invalid</mat-error>
```

Comment:
please, write that "Field is invalid. This field must contain ..." because now it can confuse users